catroidTest/src/org/catrobat/catroid/test/io/StorageHandlerTest.java

```diff
@@ -285,6 +303,37 @@ private Float interpretFormula(Formula formula, Sprite sprite) {
         return Float.NaN;
     }
 
+	public void testGetRequiredResources() throws IOException {
```

Comment:
I guess this doesn't throw an IOException...
This test should be in the ProjectTest. But other project tests aren't in the ProjectTest either... So lets stick to the "rules" and keep it here.